### ISSUE TITLE
Video broken due to listing ads.api.24ur.si in 1Hosts Pro

### ISSUE THREAD

**Issue by crssi** [badmojr/1Hosts#98]
1. Visit `https://www.24ur.com/novice/tujina/po-odpovedi-motorja-po-vsem-svetu-prizemljili-letala-boeing-777.html`
2. Start any video

What happens: Instead of the real video you can see a video which is telling you that you are using AD blocker.

The site is using `ads.api.24ur.si` just to check if AD blocker is in use... the actual ads on this site are driven over `ads.24ur.com` which is also, correctly, listed in Lite and Pro.

`ads.api.24ur.si` should be removed.

Cheers

**Comment by badmojr** [comment]
Got a video ad immediately after removing this from the list.
Is it really worth it?

**Comment by crssi** [comment]
This is a news site.
The AD doesn't come itself, you have to start/initiate the video. All intrusive ADS on this page are already covered by another listing, that is `ads.24ur.com` and that is fine.
Unfortunately nothing can be done to avoid AD stream before the actual news stream, but as said it is user initiated.
If you block `ads.api.24ur.si` the user has no choice and cannot see the news video at all.
I must not rule the user will, since this is not some harmful security issue.

On the other hand, it doesn't matter, since it looks already that the Pro is to harsh and the users will hate me if introduced.

Cheers

**Comment by badmojr** [comment]
Sorry ! This is gonna stay blocked on **Pro** since it's only purpose is anti-adblock.

**Comment by crssi** [comment]
Understood and no problem at all.

Thank you anyway.

Cheers

**Comment by badmojr** [comment]
@crssi You know wat's even funnier?
I am now able to watch the video with `ads.api.24ur.si` blocked.
How? Well, through the use/installation of the NextDNS cert as described [here](https://help.nextdns.io/t/g9hmv0a/how-to-install-and-trust-nextdns-root-ca).

**Comment by crssi** [comment]
OK. Thank you, nice to know. 😄 

But I am not interested in this method, since there are two major problems with it:
1. There are breakages and you have to be extremely careful what to block and what not, even more. Will try to find some examples later... I am sure I have seen some on community.nextdns.io or on GH issues or Reddit.
2. I cannot ask/force users to install the certificate. Not that something would be wrong with it, but they would not understand and would not like any slight idea of MITM, even if it is not harmful.

Cheers
